docs/RELEASE_NOTES.md

```diff
@@ -1,5 +1,15 @@
 # Release Notes
 
+## v4.1
+
+_January 2019:_ Version 4.1
+
+This release includes the following improvements:
+
+- Fixes some crucial bugs in twilio.js
```

Comment:
Sorry for the lack of resolution in the PR description.  Maybe something like "Fixes twilio bugs for incoming text spam and message duplicate tests"